templates/conf/kubernetes.conf.erb

```diff
@@ -20,172 +20,21 @@
   @include tail_container_parse.conf
 </source>
 
-<source>
-  @type tail
-  @id in_tail_minion
-  path /var/log/salt/minion
-  pos_file /var/log/fluentd-salt.pos
-  tag salt
-  <parse>
-    @type regexp
-    expression /^(?<time>[^ ]* [^ ,]*)[^\[]*\[[^\]]*\]\[(?<severity>[^ \]]*) *\] (?<message>.*)$/
-    time_format %Y-%m-%d %H:%M:%S
-  </parse>
-</source>
-
-<source>
-  @type tail
-  @id in_tail_startupscript
-  path /var/log/startupscript.log
-  pos_file /var/log/fluentd-startupscript.log.pos
-  tag startupscript
-  <parse>
-    @type syslog
-  </parse>
-</source>
-
-<source>
-  @type tail
-  @id in_tail_docker
-  path /var/log/docker.log
-  pos_file /var/log/fluentd-docker.log.pos
-  tag docker
-  <parse>
-    @type regexp
-    expression /^time="(?<time>[^)]*)" level=(?<severity>[^ ]*) msg="(?<message>[^"]*)"( err="(?<error>[^"]*)")?( statusCode=($<status_code>\d+))?/
-  </parse>
-</source>
-
-<source>
-  @type tail
-  @id in_tail_etcd
-  path /var/log/etcd.log
-  pos_file /var/log/fluentd-etcd.log.pos
-  tag etcd
-  <parse>
-    @type none
-  </parse>
-</source>
-
-<source>
-  @type tail
-  @id in_tail_kubelet
-  multiline_flush_interval 5s
-  path /var/log/kubelet.log
-  pos_file /var/log/fluentd-kubelet.log.pos
-  tag kubelet
-  <parse>
-    @type kubernetes
-  </parse>
-</source>
-
-<source>
-  @type tail
-  @id in_tail_kube_proxy
-  multiline_flush_interval 5s
-  path /var/log/kube-proxy.log
-  pos_file /var/log/fluentd-kube-proxy.log.pos
-  tag kube-proxy
-  <parse>
-    @type kubernetes
-  </parse>
-</source>
-
-<source>
-  @type tail
-  @id in_tail_kube_apiserver
-  multiline_flush_interval 5s
-  path /var/log/kube-apiserver.log
-  pos_file /var/log/fluentd-kube-apiserver.log.pos
-  tag kube-apiserver
-  <parse>
-    @type kubernetes
-  </parse>
-</source>
+@include kubernetes/cluster-autoscaler.conf
+@include kubernetes/containers.conf
+@include kubernetes/docker.conf
+@include kubernetes/etcd.conf
+@include kubernetes/glbc.conf
+@include kubernetes/kube-apiserver-audit.conf
+@include kubernetes/kube-apiserver.conf
+@include kubernetes/kube-controller-manager.conf
```

Comment:
Please use same order with `src`, or reorder `src`:

The order in `src`:
```
cluster-autoscaler.conf containers.conf docker.conf etcd.conf glbc.conf kube-apiserver-audit.conf kube-apiserver.conf kube-controller-manager.conf kubelet.conf kube-proxy.conf kube-scheduler.conf rescheduler.conf salt.conf startupscript.conf
```

Reply:
Resolved in c71294c1b6e11f2b6925ed45f1ec9f42dd058426

I choose to use alphabetical ascending order for both